docs/source/glossary.rst

```diff
@@ -69,131 +147,131 @@ Glossário
         <http://jats.nlm.nih.gov/faq.html#faq5>`_.
 
    ahead-of-print
-     O sistema “Ahead of Print” de publicação de artigos, disponibiliza online 
-     os trabalhos aprovados pela equipe editorial de um periódico antes mesmo 
-     da publicação impressa estar acessível aos leitores. Apesar de disponível 
-     em rede, o artigo segue o fluxo editorial de publicação impressa ou online 
-     até que esteja pronto para ser inserido em um número. O processo poderá 
+     O sistema “Ahead of Print” de publicação de artigos, disponibiliza online
```

Comment:
Ainda sobre Ahead Of Print, eu verifiquei com a Renata que está escrevendo uma documentação sobre isso, e queria saber o que vocês @robertatakenaka e @gustavofonseca acham de alterarmos a descrição de Ahead Of Print para:

"A modalidade de publicação avançada de artigos ou Ahead of Print (AOP) publica os artigos separadamente antes da composição dos números. O objetivo é contribuir para o avanço da pesquisa científica por meio da rápida comunicação dos resultados."

Essa é a maneira como consta nessa documentação (ainda em draft).